Makefile

```diff
@@ -14,14 +14,14 @@ serve:
 
 production-build: prepare
 	hugo \
-	--minify
```

Comment:
What's the reasoning behind removing minification?

Reply:
Minification sometimes messes with SVG icons and text placement, I've seen this on a few sites.

With minification, the text is closer to the SVG icons, see current https://new.goharbor.io:
![image](https://user-images.githubusercontent.com/1690215/76124475-bdc31c80-5fc8-11ea-9c88-c56cb2d2462e.png)

Without minification the spacing is better:
![image](https://user-images.githubusercontent.com/1690215/76124501-cddafc00-5fc8-11ea-9317-1140b9f2742f.png)

We can enable minification through Netlify if we want later.